index.html

```diff
@@ -176,7 +175,6 @@ <h2>Global Hospital Locator</h2>
     </div>
 
 
-  <div id="viewDiv1" data-dojo-type="dijit/layout/ContentPane" data-dojo-props="region: 'center'"></div>
```

Comment:
I am not sure why we have two view divs; I think we only need one. I don't think your use case calls for two `Map`s and two `MapView`s.